### DISCUSSION TITLE
Control multiple nodes?

### DISCUSSION THREAD

**Discussion by Kubanis** [Comfy-Org/ComfyUI#1214] [Q&A] [answered]
Is there a way to control multiple nodes (changing parameters) in once?

**Comment by hustille** [comment, marked as answer]
Depends on what you mean by 'control' ... If it's about using the same value, like an image width, in multiple places: For all nodes you want to use the value in, convert the corresponding widget to an input (right click on the node, then select 'Convert _xxx_ to input' from the context menu), create a 'Constant' node to input the value, and connect it to all the nodes. An easy way to create the input node probably is dragging out the input connector from one of the nodes onto a free spot, which should give you a menu with relevant nodes.

**Comment by Kubanis** [comment]
Thank you so much @hustille  This is excatly what I was looking for.
I had no idea it's possible even I studied ComfyUI offcial wiki and other tutorials around... Such a big help :)

**Comment by hustille** [comment]
An easy input node is utils/_Primitive_ - the nice thing is that it automagically adapts it's type to whatever you connect it to. It has some downsides, though: It can't be _Reroute_'d, and at least on my system it often refuses to connect to a second input ... In the long run you probably want to install a custom node specific to the type. Install [Manager](https://github.com/ltdrdata/ComfyUI-Manager) and have a look through the available packages.
